File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        arr=[0]*26
        for ch in s:
            index=ord(ch)-97
            arr[index]+=1

        for ch in t:
            index=ord(ch)-97
            arr[index]-=1
        
        for ele in arr:
            if ele!=0:
                return False
        return True

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] arr=new int[26];

        for(int index=0;index<s.length();index++){
            int i=(int)s.charAt(index)-97;
            arr[i]++;
        }   

        for(int index=0;index<t.length();index++){
            int i=(int)t.charAt(index)-97;
            arr[i]--;
        }

        for(int ele: arr){
            if(ele!=0){
                return false;
            }
        }

        return true;
    }
}